userguide/services.rst

```diff
@@ -1679,6 +1684,23 @@ unless there is a specific need.**
   overhead.
 
 
+Check if SMB1 support is needed by opening the :ref:`Shell` and
```

Comment:
This is a misunderstanding.  If server min protocol is set to NT1 in the file, it means the user has enabled SMB1, not that it is required.

The wording needs to be more like "Some older clients might require enabling SMB1..." and then the standard warnings and a link to the Why-Not-SMB1 article.